components/dashboard/src/settings/Account.tsx

```diff
@@ -60,31 +80,29 @@ export default function Account() {
                 title="Account"
                 subtitle="Manage account and Git configuration."
             >
-                <h3>Profile</h3>
-                <p className="text-base text-gray-500 pb-4 max-w-2xl">
-                    The following information will be used to set up Git configuration. You can override Git author name
-                    and email per project by using the default environment variables{" "}
-                    <CodeText>GIT_AUTHOR_NAME</CodeText>, <CodeText>GIT_COMMITTER_NAME</CodeText>,{" "}
-                    <CodeText>GIT_AUTHOR_EMAIL</CodeText> and <CodeText>GIT_COMMITTER_EMAIL</CodeText>.
-                </p>
-                <div className="flex flex-col lg:flex-row">
-                    <div>
-                        <div className="mt-4">
-                            <h4>Name</h4>
-                            <input type="text" disabled={true} value={user?.fullName || user?.name} />
-                        </div>
-                        <div className="mt-4">
-                            <h4>Email</h4>
-                            <input type="text" disabled={true} value={primaryEmail} />
-                        </div>
-                    </div>
-                    <div className="lg:pl-14">
-                        <div className="mt-4">
-                            <h4>Avatar</h4>
-                            <img className="rounded-full w-24 h-24" src={user!.avatarUrl} alt={user!.name} />
-                        </div>
+                <ProfileInformation
+                    profileState={profileState}
+                    setProfileState={setProfileState}
+                    errorMessage={errorMessage}
+                    updated={updated}
+                >
+                    <div className="flex flex-row mt-4">
+                        <button
+                            className="primary"
+                            disabled={!ProfileState.hasChanges(original, profileState)}
```

Comment:
**suggestion**: There's no need to disable this button. We can safely keep it enabled at all times, right? If so, shall we also change anything else related to `profileState`?

```suggestion
```